### ISSUE TITLE
Chrome extension have been deleted?

### ISSUE THREAD

**Issue by HenriqueAraujoOSF** [zertosh/jquery-audit#11]
Hello folks,

A couple days back, I noticed the extension seemed disabled, once I looked on what was happening I got the following message on my Chrome > Manage Extensions
![image](https://user-images.githubusercontent.com/56133066/117495123-9ad52500-af4b-11eb-87f7-7509d479e3dd.png)

Did the policies from Chrome Store for extensions got changed and it's no longer allowing it to be running normally?
![image](https://user-images.githubusercontent.com/56133066/117495365-e4257480-af4b-11eb-929c-048bac20f20f.png)

Seems to be removed from the store.

**Comment by HouCoder** [comment]
Here's my workaround, you can download/clone this project to your local, and then goto extensions panel in Chrome/Edge(I use Microsoft Edge), click "Load unpacked" button and select the project folder, you're good to go.

![image](https://user-images.githubusercontent.com/5292143/125225317-abb26600-e301-11eb-8ad2-9f04928c1c5b.png)

Good luck.

**Comment by lon9man** [comment]
@HouCoder yes, i did it the same way. but main question was "Why it was removed from the store?".
recently it was similar problem with extension "The Great Suspender", in which was added malware by new owner. after that extension was removed from the store, forked and cleaned by community (https://github.com/greatsuspender/thegreatsuspender/issues/1304)
